nops-aws-account-register/README.md

```diff
@@ -14,6 +14,9 @@
 ### cloudformation-single-acc-register
 This folder contains CloudFormation template to register a single AWS account with nOps.
 
+### cloudformation-org-accounts-register
+This folder contains CloudFormation template to register AWS accounts with consolidated billing and organisation to nOps.
```

Comment:
This folder contains the CloudFormation template to register AWS accounts with consolidated billing and organization to nOps.